### PR TITLE
[improve][doc] Update some auth settings in worker service

### PR DESCRIPTION
**Pull request by RobertIndie**
Master Issue: #7314 

### Motivation

https://github.com/apache/pulsar/pull/7424 has renamed some settings to make those broker-client TLS settings clearer. But some doc hasn't been updated. This PR updates these settings in the worker service. We need to update this doc in and after version 2.7.0.
### Modifications

* Replace `clientAuthenticationParameters` with `brokerClientAuthenticationParameters`
* Replace `clientAuthenticationPlugin` with `brokerClientAuthenticationPlugin`

### Documentation

- [x] `doc`

### CODE REVIEW

File: site2/docs/functions-worker.md [hunk 1]
```diff
@@ -124,8 +124,9 @@ To run function-worker separately, you have to configure the following parameter
 
 If authentication is enabled for your broker cluster, you *should* configure the authentication plugin and parameters for the functions worker to communicate with the brokers.
 
-- `clientAuthenticationPlugin`
-- `clientAuthenticationParameters`
+- `brokerClientAuthenticationEnabled`: Whether to enable the broker client authentication used by function workers to talk to brokers.
+- `clientAuthenticationPlugin`: The authentication plugin to be used by the pulsar client used in worker service.
```
Comment (Anonymitaet) [inline site2/docs/functions-worker.md:128]:
```suggestion
- `clientAuthenticationPlugin`: The authentication plugin to be used by the Pulsar client used in worker service.
```

File: site2/docs/functions-worker.md [hunk 2]
```diff
@@ -124,8 +124,9 @@ To run function-worker separately, you have to configure the following parameter
...
+- `clientAuthenticationParameters`: The authentication parameter to be used by the pulsar client used in worker service.
```
Comment (Anonymitaet) [inline site2/docs/functions-worker.md:129]: same

**Comment by RobertIndie** [comment]
@Anonymitaet Thanks! I have updated the PR. PTAL.

**Review by Anonymitaet — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@RobertIndie I think you need to rebase to the master branch

**Pull request merged**